hbase_sql/src/main/java/com/splicemachine/derby/stream/spark/SparkDataSetProcessor.java

```diff
@@ -353,20 +357,43 @@ public StructType getExternalFileSchema(String storedAs, String location){
 
             }
             if (storedAs.toLowerCase().equals("t")) {
-                schema =  SpliceSpark.getSession().read().csv(location).schema();
+                 schema = getCsvSchema(location);
             }
         }
 
         return schema;
     }
 
+
+    private StructType getCsvSchema(String location) throws IOException {
+        InputStream is = null;
```

Comment:
Agree, the csv file may not be produced by splice with schema information. Let's bypass schema checking for csv file